test/e2e/suite/issuers/acme/certificate/http01.go

```diff
@@ -258,17 +256,16 @@ var _ = framework.CertManagerDescribe("ACME Certificate (HTTP01)", func() {
 			Fail("Neither " + networkingv1.SchemeGroupVersion.String() + " nor " + networkingv1beta1.SchemeGroupVersion.String() + " were discovered in the API server")
 		}
 
-		certClient := f.CertManagerClientSet.CertmanagerV1().Certificates(f.Namespace.Name)
 		By("Waiting for Certificate to exist")
-		err := util.WaitForCertificateToExist(certClient, certificateSecretName, foreverTestTimeout)
+		cert, err := f.Helper().WaitForCertificateToExist(f.Namespace.Name, certificateSecretName, time.Second*60)
```

Comment:
Here again, we seem to be using the name of the Secret as the name of the Certificate. 
```suggestion
		cert, err := f.Helper().WaitForCertificateToExist(f.Namespace.Name, certificateName, time.Second*60)
```

I guess it works because both names are the same. 

I can't face another round of code review and E2E tests. Let's just leave it.